poll-webapp/src/main/webapp/skin/less/poll.less

```diff
@@ -34,4 +34,128 @@
         padding: 5px 5px;
       }
   }
+}
+
+#poll-activity {
```

Comment:
Format the file correctly and transform css classes to an appropriate less classes tree